.github/workflows/build.yml

```diff
@@ -245,7 +245,7 @@ jobs:
       matrix:
         arch: ["arm", "arm64", "386", "amd64"]
     env:
-      repo: ${{ github.event.repository.name }}
+      repo: "${{ env.PKG_NAME }}"
```

Comment:
Thanks, Sam! I just updated this to use `env.PKG_NAME` instead of `env.repo` for the docker build job. Nightly releases aren't in use by any other teams yet -- this was created specifically for packer to have feature parity with existing code in packer-releases -- but there's an open PR in the HelloWorld project discussing adding this support across products. Feel free to comment there if you have suggestions!